src/pykeen/models/unimodal/rescal.py

```diff
@@ -51,7 +59,12 @@ def __init__(
         random_seed: Optional[int] = None,
         regularizer: Optional[Regularizer] = None,
     ) -> None:
-        """Initialize the model."""
+        """Initialize the model.
```

Comment:
No documentation of the constructor parameters.

Reply:
Addressed in 72180ab5